src/envo/envo-edit.owl

```diff
@@ -31615,76 +31633,133 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001406> ObjectSomeValuesFrom(<
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001411> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/ENVO_03520001>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001411> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000086> <http://purl.obolibrary.org/obo/PATO_0002124>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001411> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002473> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/BFO_0000027> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002351> ObjectUnionOf(<http://purl.obolibrary.org/obo/ENVO_01000833> <http://purl.obolibrary.org/obo/ENVO_03520006> <http://purl.obolibrary.org/obo/ENVO_03520007>)))))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001411> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002473> ObjectUnionOf(<http://purl.obolibrary.org/obo/ENVO_00002006> <http://purl.obolibrary.org/obo/ENVO_01000277>)))
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_01001412> (stratocumulus cloud)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://cloudatlas.wmo.int/clouds-genera-stratocumulus.html") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://cloudatlas.wmo.int/clouds-genera.html") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://cloudatlas.wmo.int/docs/Excerpt%20from%20wmo_49-1_final_2017.pdf") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Cloud") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01001412> "A cloud which is 1) grey or whitish, or both grey and whitish in colour, 2) structurally organised as a patch, sheet or layer that a) almost always has dark parts and b) is composed of non-fibrous tessellations, rounded masses, rolls, and similar structural elements and which may or may not be merged.")
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://cloudatlas.wmo.int/clouds-genera-stratocumulus.html") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://cloudatlas.wmo.int/clouds-genera.html") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://cloudatlas.wmo.int/docs/Excerpt%20from%20wmo_49-1_final_2017.pdf") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Cloud") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01001412> "A cloud which 1) has been shaped into non-fibrous tessellations, rounded masses, rolls, and similar structural elements which may or may not be merged, 2) is composed of water droplets, sometimes accompanied by raindrops or, more rarely, snow pellets, snow crystals and snowflakes, 3) is located in the low level of the atmosphere (low étage)."@en)
```

Comment:
>which may or may not be merged

One may (or may not) need to say this 😉 

>and similar structural elements

The bounds of this similarity could be an issue - not something we can resolve, but perhaps something to put into the comment. 

>is composed of water droplets, sometimes accompanied by raindrops or, more rarely, snow pellets, snow crystals and snowflakes,


Consider rephrasing to: "...is primarily composed of water droplets and may contain raindrops, snow pellets, snow crystals, or snowflakes..."

Reply:
> One may (or may not) need to say this 😉

😄 Deleted the phrase.

> The bounds of this similarity could be an issue - not something we can resolve, but perhaps something to put into the comment.

Moved that section into the comment.

> Consider rephrasing to: "...is primarily composed of water droplets and may contain raindrops, snow pellets, snow crystals, or snowflakes..."

✔️